### ISSUE TITLE
Check first if given pull request is closed in `github checkout-pr`

### ISSUE THREAD

**Issue by rifel123** [VirtusLab/git-machete#227]
Github Api returns all opened pull requests after `GET` on [pulls endpoint](https://docs.github.com/en/rest/reference/pulls).

During `github checkout-pr` on pull request which is already closed, git-machete ends up with message:
```
$ git machete github checkout-pr 204

PR number 204 is not found in repository git-machete
```
Scope of this issue is to make distinction in the error message when pull request is closed or never created.

**Comment by PawelLipski** [comment]
A test for both cases (non-existent vs closed PR) should be included as well.

**Comment by PawelLipski** [comment]
Actually... why forbid checking out closed PRs? 🤔 I'd say let the user check them out, just with a warning maybe

**Comment by rifel123** [comment]
closed with #248
